consensus/scp/core/src/core_types.rs

```diff
@@ -125,7 +125,7 @@ impl<V: Value> PartialOrd for Ballot<V> {
 // This makes debugging easier when looking at large ballots.
 impl<V: Value> fmt::Display for Ballot<V> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let mut hasher = DefaultHasher::new();
+        let mut hasher = HasherBuilder::default().build_hasher();
```

Comment:
There's not specifically good reason to use HasherBuilder anywhere we can. It's uses specific qualities which are useful in the enclave but it's not necessarily the best default choice everywhere.

Reply:
The main reason here is that `DefaultHasher` is not `no_std` compatible, and `HasherBuilder` is a compatible impl within reach. This is in `Display` so we don't have to maintain backwards compatibility.